onfido-java/src/test/java/com/onfido/integration/AddressManagerTest.java

```diff
@@ -0,0 +1,46 @@
+package com.onfido.integration;
+
+import com.onfido.JsonObject;
+import com.onfido.Onfido;
+import com.onfido.models.Address;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class AddressManagerTest extends ApiIntegrationTest {
+
+    @Test
+    public void pickTest() throws Exception {
+        String addresss1 = new JsonObject()
+                .add("postcode", "postcode1").toJson();
+
+        String addresss2 = new JsonObject()
+                .add("postcode", "postcode2").toJson();
+
+        String response = new JsonObject().add("addresses", new String[]{addresss1, addresss2}).toJson()
+                .replace("\\", "")
+                .replace("\"{", "{")
+                .replace("}\"", "}");
```

Comment:
Wouldn't mind adding a `JsonArray` class similar to `JsonObject`, it would make these tests easier to read/understand.

Something that would allow e.g.

```java
String response = new JsonArray(
    new JsonObject().add("postcode", "postcode1")),
    new JsonObject().add("postcode", "postcode2"))
).toJson()
```